plaso/engine/path_helper.py

```diff
@@ -408,3 +415,47 @@ def GetWindowsSystemPath(cls, path, environment_variables):
     path = cls.ExpandWindowsPath(path, environment_variables)
 
     return path, filename
+
+  @classmethod
+  def SanitizePathSegments(cls, path_segments):
```

Comment:
I would recommend we move this to the cli submodule and only use it for CLI and log output.

Reply:
will wait for you reply to the comment above regarding comparing path of extracted artifact to the sanitized version of the path extracted from the trie as well. If we remove this, I will move this back to the cli sub module. but moving it there now will introduce cyclic import as cli module import artifact_trie which needs to access this method

Reply:
I see, let's leave it in for now